### PR TITLE
feat(tpu): add tpu  queued resources list sample

### PR DESCRIPTION
**Pull request by TetyanaYahodska**
## Description
Documentation - https://cloud.google.com/tpu/docs/queued-resources
Sample in Python - https://github.com/GoogleCloudPlatform/python-docs-samples/pull/12716/files
Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist

- [x] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md)
- [ ] `pom.xml` parent set to latest `shared-configuration`
- [ ] Appropriate changes to README are included in PR
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] **Tests** pass:   `mvn clean verify` **required**
- [x] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [ ] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample 
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: tpu/src/main/java/tpu/ListQueuedResources.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    // Project ID or project number of the Google Cloud project.
+    String projectId = "YOUR_PROJECT_ID";
+    // The zone in which the TPU was created.
+    String zone = "us-central1-a";
+
+    listQueuedResources(projectId, zone);
+  }
+
+  // List Queued Resources.
+  public static TpuClient.ListQueuedResourcesPage listQueuedResources(
```
Comment (Sita04) [inline tpu/src/main/java/tpu/ListQueuedResources.java:36]: Import the `TpuClient.ListQueuedResourcesPage` and only specify `ListQueuedResourcesPage` as the return type.

File: tpu/src/main/java/tpu/ListQueuedResources.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      String projectId, String zone) throws IOException {
+    String parent = String.format("projects/%s/locations/%s", projectId, zone);
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (TpuClient tpuClient = TpuClient.create()) {
+      ListQueuedResourcesRequest request =
+          ListQueuedResourcesRequest.newBuilder().setParent(parent).build();
+
+      return tpuClient.listQueuedResources(request).getPage();
```
Comment (Sita04) [inline tpu/src/main/java/tpu/ListQueuedResources.java:45]: Print the list of resources here and then return the value.

File: tpu/src/test/java/tpu/QueuedResourceIT.java [hunk 3]
```diff
@@ -120,6 +124,33 @@ public void testGetQueuedResource() throws IOException {
...
+  public void testListTpuVm() throws IOException {
+    try (MockedStatic<TpuClient> mockedTpuClient = mockStatic(TpuClient.class)) {
+      QueuedResource queuedResource1 = mock(QueuedResource.class);
+      QueuedResource queuedResource2 = mock(QueuedResource.class);
+      List<QueuedResource> mockListQueuedResources =
+          Arrays.asList(queuedResource1, queuedResource2);
+
+      TpuClient mockClient = mock(TpuClient.class);
+      mockedTpuClient.when(TpuClient::create).thenReturn(mockClient);
+      TpuClient.ListQueuedResourcesPagedResponse mockListQueuedResourcesResponse =
```
Comment (Sita04) [inline tpu/src/test/java/tpu/QueuedResourceIT.java:137]: Same comment here. Import the class and simplify the data type.

**Review by Sita04 — CHANGES_REQUESTED** [review]

File: tpu/src/main/java/tpu/ListQueuedResources.java (hunk 1 above)
Comment (TetyanaYahodska) [reply tpu/src/main/java/tpu/ListQueuedResources.java:36]: Thank you! Fixed

File: tpu/src/main/java/tpu/ListQueuedResources.java (hunk 2 above)
Comment (TetyanaYahodska) [reply tpu/src/main/java/tpu/ListQueuedResources.java:45]: Thank you! Added

File: tpu/src/test/java/tpu/QueuedResourceIT.java (hunk 3 above)
Comment (TetyanaYahodska) [reply tpu/src/test/java/tpu/QueuedResourceIT.java:137]: Thank you1 Fixed

**Review by Sita04 — APPROVED** [review]

**Pull request merged**